rp2040-hal/src/usb.rs

```diff
@@ -256,7 +256,7 @@ impl Inner {
             // size in 64bytes units.
             // NOTE: the compiler is smart enough to recognize /64 as a 6bit right shift so let's
             // keep the division here for the sake of clarity
```

Comment:
This is not true any more.

Reply:
Looks like still is. Or do you think it's confusing that the comment says `/64` and not `div_ceil(64)`? Both are divisions.